server/controllers/stock/reports/stock_value.report.handlebars

```diff
@@ -4,61 +4,56 @@
-        {{translate 'REPORT.STOCK_VALUE.TITLE'}}
-      </h2>
-
-      <h3 class="text-center">
-        {{#if depot.text}}{{depot.text}}{{/if}}
-      </h3>
-
-      <h4 class="text-center">
-        {{date dateTo}}
-      </h4>
-
-      <br>
-
-      <!-- list of data  -->
-      <table class="table table-condensed table-report">
-        <thead>
-          <tr style="background-color:#ddd;">
-            <th class="text-center">{{translate 'FORM.LABELS.INVENTORY'}}</th>
-            <th class="text-center">{{translate 'FORM.LABELS.QUANTITY'}}</th>
-            <th class="text-center">{{translate 'STOCK.UNIT_COST'}}</th>
-            <th class="text-center">{{translate 'FORM.LABELS.VALUE'}}</th>
-          </tr>
-        </thead>
-        <tbody>
-          {{#each stockValues}}
-            <tr>
-              <td style="width:50%">{{inventory_name}}</td>
-              <td class="text-right">{{stockQtt}}</td>
-              <td class="text-right">{{stockUnitCost}}</td>
-              <td class="text-right">{{currency stockValue ../currency_id}}</td>
+    <!-- body  -->
+    <div class="row">
+      <div class="col-xs-12">
+
+        <!-- page title  -->
+        <h2 class="text-center text-uppercase">
+          {{translate 'REPORT.STOCK_VALUE.TITLE'}}
+        </h2>
+
+        <h3 class="text-center">
+          {{#if depot.text}}{{depot.text}}{{/if}}
+        </h3>
+
+        <h4 class="text-center">
+          {{date dateTo}}
+        </h4>
+
+        <br>
+
+        <!-- list of data  -->
+        <table class="table table-condensed table-report">
+          <thead>
+            <tr style="background-color:#ddd;">
+              <th class="text-center">{{translate 'FORM.LABELS.INVENTORY'}}</th>
+              <th class="text-center">{{translate 'FORM.LABELS.QUANTITY'}}</th>
+              <th class="text-center">{{translate 'STOCK.UNIT_COST'}}</th>
+              <th class="text-center">{{translate 'FORM.LABELS.VALUE'}}</th>
             </tr>
-          {{/each}}
-           <!-- no data -->
-          {{#if emptyResult}}
-            <tr><th class="text-center" colspan="4">{{translate 'STOCK.NO_DATA'}}</th></tr>
-          {{else}}
-            <tr>
-              <th colspan="3" class="text-right">{{ translate "FORM.LABELS.TOTAL"}}</th>
-              <th class="text-right">{{ currency stocktotal ./currency_id }}</th>
-            </tr>
-          {{/if}}
-
-
-        </tbody>
-
-      </table>
+          </thead>
+          <tbody>
+            {{#each stockValues}}
+              <tr>
+                <td style="width:50%">{{inventory_name}}</td>
+                <td class="text-right">{{stockQtt}}</td>
+                <td class="text-right">{{currency (multiply stockUnitCost ../rate) ../currency_id}}</td>
+                <td class="text-right">{{currency (multiply stockValue ../rate) ../currency_id}}</td>
+              </tr>
+            {{/each}}
+            <!-- no data -->
+            {{#if emptyResult}}
```

Comment:
There is no need for this "emptyResult" check.  Just use `{{#each}}{{else}}{{/each}}` as discussed in WhatsApp.

So, it would look like this:
```hbs
{{#each stockValues}}
   {{! ... etc ... }}
{{else}}
  <tr><th class="text-center" colspan="4">{{translate 'STOCK.NO_DATA'}}</th></tr>
{{/each}}
```